### PR TITLE
Add testing sweep through concrete thermal/moisture properties

### PR DESCRIPTION
**Pull request by bwspenc**
ref #216

### CODE REVIEW

File: test/tests/concrete_moisture_heat_transfer/concrete_thermal_moisture_properties.i [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+[]
+
+[Problem]
+  solve = false
+[]
+
+[AuxVariables]
+  [temperature]
+    order = CONSTANT
+    family = MONOMIAL
```
Comment (jain651) [inline test/tests/concrete_moisture_heat_transfer/concrete_thermal_moisture_properties.i:15]: We need to provide initial condition, else I am getting mismatched value for the material properties at the initial time step.

`initial_condition = 20`

File: test/tests/concrete_moisture_heat_transfer/concrete_thermal_moisture_properties.i [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+  []
+  [rh]
+    order = CONSTANT
+    family = MONOMIAL
+  []
```
Comment (jain651) [inline test/tests/concrete_moisture_heat_transfer/concrete_thermal_moisture_properties.i:20]: The same comment here.

`initial_condition = 1`

File: test/tests/concrete_moisture_heat_transfer/concrete_thermal_moisture_properties.i [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+[]
+
+[AuxVariables]
+  [temperature]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [rh]
+    order = CONSTANT
+    family = MONOMIAL
```
Comment (jain651) [inline test/tests/concrete_moisture_heat_transfer/concrete_thermal_moisture_properties.i:19]: The same comment here.

`initial_condition = 1`

**Review by jain651 — COMMENTED** [review]
We do not need `concrete_thermal_moisture_properties_out.csv` file in the gold directory.

**Review by jain651 — CHANGES_REQUESTED** [review]
Do we want to keep `_moisture_capacity[_qp]` term in `ConcreteMoistureTimeIntegration.C`?

File: src/materials/ConcreteThermalMoisture.C [hunk 4]
```diff
@@ -59,15 +58,14 @@ ConcreteThermalMoisture::validParams()
...
       "moisture_diffusivity_model", moisture_diffusivity_model, "moisture diffusivity models");
 
   params.addParam<Real>("cement_mass", 354.0, "cement mass (kg) per m^3");
   params.addParam<Real>("aggregate_mass", 1877.0, "aggregate mass (kg) per m^3");
   params.addParam<Real>("water_to_cement_ratio", 0.43, "water to cement ratio");
   params.addParam<Real>("concrete_cure_time", 23.0, "concrete curing time in days");
-  params.addParam<Real>("aggregate_vol_fraction", 0.7, "volumetric fraction of aggregates");
 
   // parameters for Bazant mositure diffusivity model
   params.addParam<Real>("D1", 3.0e-10, "empirical constants");
```
Comment (jain651) [inline src/materials/ConcreteThermalMoisture.C:71]: We can add unit in the parameter description. It should be m^2/sec

Comment (bwspenc) [reply src/materials/ConcreteThermalMoisture.C:71]: I agree with both of these comments, but I think they're both better handled in separate PRs. I was trying to focus on testing the material models and deleting the ones that have issues in this one. We still don't have testing of the PDE solution in this PR, but I was planning on a follow-on PR that removes the moisture capacity from `ConcreteMoistureTimeIntegration` and adds a revised version of the Maqbeth test. I was assuming the documentation cleanup (including parameter descriptions) would be handled in your PR.

Comment (jain651) [reply src/materials/ConcreteThermalMoisture.C:72]: Okay. I will submit a PR for these changes once this is merged.

**Review by jain651 — APPROVED** [review]

**Review by dschwen — APPROVED** [review]

**Pull request merged**